src/azure/managementUrlHelper.ts

```diff
@@ -0,0 +1,53 @@
+import * as core from '@actions/core';
+import * as exec from '@actions/exec';
+import * as io from "@actions/io";
+
+export class ManagementUrlHelper{
+
+    public static async getBaseUrl(): Promise<string> {
+        if(!this._baseUrl) {            
+            let azCloudDetails = JSON.parse(await ManagementUrlHelper.executeAzCliCommand('cloud show'));
+            const cloudEndpoints = azCloudDetails['endpoints'];
+            this._baseUrl = ManagementUrlHelper.getResourceManagerUrl(cloudEndpoints);
+        }
+
+        return this._baseUrl;
+    }
+    
+    public static async executeAzCliCommand(command: string, args?: string[]): Promise<string> {
```

Comment:
This method looks duplicate of `getTokenFromAzCLI()` in `azAuthentication`. Suggestion:
- Make this class `AzCli`
- Make this method `executeCommand`
- Remove `getTokenFromAzCLI` from `azAuthentication` and use `AzCli.azAuthentication()`

Reply:
Created AzCli Class. Moved getAccessToken method here as well